templates/focus-ios.xliff

```diff
@@ -829,6 +865,26 @@
         <source>Site missing content or acting strange?</source>
         <note>Text for a label that indicates the title for sites not working tip.</note>
       </trans-unit>
+      <trans-unit id="TooltipBodyText.ContextMenu">
+        <source>Go to Settings to manage specific privacy &amp; security options.</source>
+        <note>This is the body text that is displayed for the Context Menu icon tooltip</note>
+      </trans-unit>
+      <trans-unit id="TooltipBodyText.Privacy">
+        <source>These default settings offer strong protection. But it’s easy to tweak the settings to meet your specific needs.</source>
+        <note>This is the body text that is displayed for the Privacy tooltip</note>
+      </trans-unit>
+      <trans-unit id="TooltipBodyText.ShieldIcon">
+        <source>Focus stopped this site from spying on you. Tap the shield for info on what we blocked.</source>
```

Comment:
Placeholder for app name

Reply:
corrected here: https://github.com/mozilla-l10n/focusios-l10n/pull/80/files#diff-32781eb8e0a5f13c0370cf3bf8653e7bb3a02d5269b57edfc8266c10d272c7b3R1050-R1052